components/button/button-icon.js

```diff
@@ -84,7 +85,7 @@ class ButtonIcon extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement)
 				:host([active]) button {
 					background-color: var(--d2l-color-gypsum);
 				}
-				button:focus {
+				button.focus-visible {
```

Comment:
This is failing our CSS linting with:
> Expected class selector ".focus-visible" to match specified pattern   selector-class-pattern

So I guess I'll need to add an exception -- probably best to do it in the global stylelint-config eh @dbatiste?

Reply:
Created a PR to add it:
https://github.com/BrightspaceUI/stylelint-config/pull/1

Reply:
Yeah, the alternative would be that we disable the rule with inline comments, or remove that rule entirely... but adding it seems good!